package.json

```diff
@@ -15,18 +15,18 @@
   "scripts": {
     "postinstall": "manypkg check && preconstruct dev && yarn --cwd docs",
     "auth": "npm_config_registry=https://registry.npmjs.org npm whoami",
-    "clean": "lerna exec 'rm -rf dist'",
+    "clean": "manypkg exec 'rm -rf dist'",
     "generate-readmes": "yarn generate-readme --all-workspace-packages",
-    "generate-icons": "lerna run generate-icons && prettier --write '**/generated/*.tsx'",
-    "design-tokens:build": "yarn --cwd design-system build:tokens",
-    "design-tokens:build:watch": "yarn --cwd design-system build:tokens:watch",
+    "generate-icons": "manypkg run icons generate-icons && manypkg run checkbox-input generate-icons && manypkg run radio-input generate-icons && manypkg run @commercetools-uikit/rich-text-input generate-icons && prettier --write '**/generated/*.tsx'",
```

Comment:
Same as app-kit. We're explicit here and have to. There is an issue to manypkg opened which would allow globbing or running a run script based on its existence.